tests/functional/spec/features/dynamic_answer_options/radio_options_from_checkbox_answers.js

```diff
@@ -0,0 +1,41 @@
+import HealedTheQuickestPage from "../../../generated_pages/dynamic_radio_options_from_checkbox/healed-the-quickest.page";
+import InjurySustainedPage from "../../../generated_pages/dynamic_radio_options_from_checkbox/injury-sustained.page";
+import MostSeriousInjuryPage from "../../../generated_pages/dynamic_radio_options_from_checkbox/most-serious-injury.page";
+import SubmitPage from "../../../generated_pages/dynamic_radio_options_from_checkbox/submit.page";
+
+describe("Dynamic radio options from checkbox answers", () => {
+  describe("Given the dynamic radio options from checkbox questionnaire and I am on the checkbox answer page", () => {
+    it("When the respondent answers the checkbox question and submits, Then the radio question should show the answers from that checkbox as options, as well as a static option", () => {
+      browser.openQuestionnaire("test_dynamic_radio_options_from_checkbox.json");
+      $(InjurySustainedPage.head()).click();
+      $(InjurySustainedPage.body()).click();
+      $(InjurySustainedPage.submit()).click();
+
+      expect(browser.getUrl()).to.contain(MostSeriousInjuryPage.pageName);
+      expect($(MostSeriousInjuryPage.answerLabelByIndex(0)).getText()).to.contain("Head");
+      expect($(MostSeriousInjuryPage.answerLabelByIndex(1)).getText()).to.contain("Body");
+      expect($(MostSeriousInjuryPage.answerLabelByIndex(2)).getText()).to.contain("They were of equal severity (static option)");
+      expect($(MostSeriousInjuryPage.answerLabelByIndex(3)).isExisting()).to.be.false;
+    });
+
+    it("When the respondent answers the radio question and submits, Then the radio question should show only the answers from the first checkbox as options", () => {
```

Comment:
Might be clearer if it said `Then the next radio question..`, found it confusing as you are on a radio, so made me this it was somehow still the same radio.